docs/talks/intro-api/talk.Rmd

```diff
@@ -0,0 +1,349 @@
+
+What does the current COVID-19 cumulative case rate look like, nationwide?
+
+```{r, fig.width=9, fig.height=7}
+cases = covidcast_signal(data_source = "jhu-csse", 
+                         signal = "confirmed_cumulative_prop",
+                         start_day = "2020-08-30")
+
+plot(cases, title = "Cumulative COVID-19 cases per 100,000 people", 
+     choro_params = list(legend_n = 6))
+```
+
+# Example: Cases (Cont.)
+
+Where is the current COVID-19 cumulative case rate greater than 2%?
+
+```{r, fig.width=9, fig.height=7}
+plot(cases, choro_col = c("#D3D3D3", "#FFC0CB"), 
+     title = "Cumulative COVID-19 cases per 100,000 people",
+     choro_params = list(breaks = c(0, 2000), legend_width = 5))
+```
+
+# Example: Doctor's Visits
+
+How do some major cities compare in terms of doctor's visits due to COVID-like illness?
+
+```{r, fig.width=7, fig.height=5}
+dv = covidcast_signal(data_source = "doctor-visits", 
+                      signal = "smoothed_adj_cli", 
+                      start_day = "2020-03-01", geo_type = "msa", 
+                      geo_values = name_to_cbsa(c("Pittsburgh", "New York", 
+                                                  "San Antonio", "Miami")))
+
+plot(dv, plot_type = "line", 
+     title = "% of doctor's visits due to COVID-like illness")
+```
+
+# Example: Symptoms
+
+How do my county and my friend's county compare in terms of people reporting that they know somebody with COVID symptoms?
+
+```{r, fig.width=7, fig.height=5}
+sympt = covidcast_signal(data_source = "fb-survey", 
+                         signal = "smoothed_hh_cmnty_cli",
+                         start_day = "2020-04-15", 
+                         geo_values = c(name_to_fips("Allegheny"),
+                                        name_to_fips("Fulton", state = "GA")))
+
+plot(sympt, plot_type = "line", range = range(sympt$value),
+     title = "% of people who know somebody with COVID symptoms")
+```
+
+# API Specification
+
+The COVIDcast API is based on HTTP GET queries and returns data in JSON form. The base URL is https://delphi.cmu.edu/epidata/api.php?source=covidcast
+
+<br>
+
+| Parameter | Description | Examples |
+| :-- | :-- | :-- |
+| `data_source` | data source | `doctor-visits` or `fb-survey` |
+| `signal` | signal derived from data source | `smoothed_cli` or `smoothed_adj_cli` |
+| `time_type` | temporal resolution of the signal | `day` or `week` |
+| `geo_type` | spatial resolution of the signal | `county`, `hrr`, `msa`, or `state` |
+| `time_values` | time units over which events happened | `20200406` or `20200406-20200410` |
+| `geo_value` | location codes, depending on `geo_type` | `*` for all, or `pa` for Pennsylvania |
+
+# Example: API Query
+
+Estimated % COVID-like illness on April 6, 2020 from the Facebook survey, in Allegheny County: https://delphi.cmu.edu/epidata/api.php?source=covidcast&data_source=fb-survey&signal=raw_cli&time_type=day&geo_type=county&time_values=20200406&geo_value=42003 
```

Comment:
Also should change this example, and the code, to use `api.covidcast.cmu.edu`

Reply:
Thanks, fixed!